src/components/input/Input.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import input from './Input.module.css';
+
+function Input() {
+  return (
+            <div className={input.input_sms}>
+            <input
```

Comment:
add basic validation. fe button is disabled when the length of the text in the input is 0